core/src/cluster_info.rs

```diff
@@ -356,7 +356,7 @@ pub fn make_accounts_hashes_message(
 }
 
 // TODO These messages should go through the gpu pipeline for spam filtering
-#[frozen_abi(digest = "3ZHQscZ9SgxKh45idzHv3hiagyyPRtDgeySmJn171PTi")]
```

Comment:
What was the cause of this ABI change?

Reply:
I also [hit this](https://buildkite.com/solana-labs/solana/builds/33364#ebba9083-953f-4c86-a60a-a25d79f88eda/568-1211) when I "resolved" a merge conflict in `Cargo.lock` by deleting it and regenerating with `cargo tree`.  It turns out that doing so results in much broader changes than `git checkout Cargo.lock; cargo tree`.  I suspect some dependency's ABI changed and propagated to our types.

You might try resetting `Cargo.lock` to the base and regenerating it

Reply:
Thx, solved all cases of `#[frozen_abi(digest = `